aten/src/ATen/TensorSubclassLikeUtils.h

```diff
@@ -39,16 +40,22 @@ constexpr auto kTensorSubclassLike =
     DispatchKeySet(BackendComponent::MetaBit);
 
 inline bool isTensorSubclassLike(const Tensor& tensor) {
+  if (c10::impl::dispatch_mode_enabled())
+    return true;
```

Comment:
nit: parens please

Reply:
The loss of parens is clang format enforced 😂. I wonder if this formatting setting is configurable